participants.py

```diff
@@ -0,0 +1,12 @@
+import pandas as pd
+import unidecode
+
+df = pd.read_excel("input/H21 - Liste complète.xlsx", header=8, usecols="B:F")
+df2 = pd.read_excel("input/AGEGListeComplèteA20.xlsx", header=8, usecols="B:F")
+participants = pd.DataFrame(columns=["email", "lastname", "firstname"], index=df.index)
+participants["lastname"] = df["Nom choisi"]
+participants["firstname"] = df["prénom choisi"]
```

Comment:
```suggestion
participants["firstname"] = df["Prénom choisi"]
```

Sa serait pas plus uniforme comme cela ?

Reply:
C'est simplement le nom de la colonne dans le fichier fourni. Idéalement, je préfère ne pas trop toucher aux fichiers en input.